modules/core/src/main/java/org/apache/ignite/cache/store/jdbc/dialect/OracleDialect.java

```diff
@@ -28,6 +28,12 @@
     /** */
     private static final long serialVersionUID = 0L;
 
+    private static final String MERGE_ON = "MERGE INTO %s t USING (SELECT %s FROM dual) v ON %s";
```

Comment:
according to [coding guidelines](https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines#CodingGuidelines-JavadocComments) javadoc is mandatory